pkg/utils/factory.go

```diff
@@ -363,3 +364,29 @@ func NewClusterRoleBinding(bindingName, roleName string, subjects []rbac.Subject
 		Subjects: subjects,
 	}
 }
+
+func ServiceMonitor(serviceMonitorName, namespace string) *monitoringv1.ServiceMonitor {
+	return &monitoringv1.ServiceMonitor{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       monitoringv1.ServiceMonitorsKind,
+			APIVersion: monitoringv1.Group + "/" + monitoringv1.Version,
```

Comment:
Can this be replaced with `monitoringv1.SchemeGroupVersion.String()` Ref: https://github.com/openshift/cluster-logging-operator/blob/master/pkg/utils/factory.go#L21